tests/unit/Lookup/RestrictedEntityLookupTest.php

```diff
@@ -37,7 +38,10 @@ private function getEntityLookup() {
 
 	public function testConstructor() {
 		$lookup = new RestrictedEntityLookup( $this->getEntityLookup(), 1 );
-		$this->assertInstanceOf( RestrictedEntityLookup::class, $lookup );
+		$this->assertInstanceOf(
+			'Wikibase\DataModel\Services\Lookup\RestrictedEntityLookup',
```

Comment:
I was about to suggest using ::class and then saw you actually moved away from it. I am confoose.

Reply:
Oh, It was a rebase mistake. Let me fix it.